docker/configFile.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
The problem is that this is not available at build time, only the files inside the docker folder are included in the build context. (As far as I understand it anyway).

Maybe @linuxbandit has a better solution for this?